external/storm-redis/src/main/java/org/apache/storm/redis/common/config/JedisClusterConfig.java

```diff
@@ -45,6 +45,26 @@
      * @param nodes list of node information for JedisCluster
      * @param timeout socket / connection timeout
      * @param maxRedirections limit of redirections - how much we'll follow MOVED or ASK
+     * @throws NullPointerException when you didn't set nodes
+     */
+    public JedisClusterConfig(Set<InetSocketAddress> nodes, int timeout, int maxRedirections) {
+        Preconditions.checkNotNull(nodes, "Node information should be presented");
```

Comment:
Maybe better to call `this(nodes, timeout, maxRedirections, null);` to remove duplicate codes.